plugins/action/cp_mgmt_access_layers.py

```diff
@@ -81,7 +80,7 @@ def search_for_resource_name(self, conn_request, payload):
             search_result = self.search_for_existing_rules(
                 conn_request,
                 self.api_call_object_plural_version,
-                search_payload,
+                {"offset": 0},
```

Comment:
Isn't necessary since this is the default value when running the API command, and the search_payload here is an empty dict which is fine and worked for me:)